Perlang.Common/IInterpreter.cs

```diff
@@ -4,8 +4,6 @@ namespace Perlang
 {
     public interface IInterpreter
     {
-        List<string> Arguments { get; }
-
         void ExecuteBlock(IEnumerable<Stmt> statements, IEnvironment blockEnvironment);
```

Comment:
Given that we only have this single method left here now, we could possible get rid of this interface altogether. I might give it a try in a subsequent PR.